### PR TITLE
Update wrong placeholders in ar-arb translation

### PR DESCRIPTION
**Pull request by quintanilhar**
## Ticket
https://oat-sa.atlassian.net/browse/TR-3026

## Summary
Replacement characters (%s, %d, …) are not replaced in translations strings rendered.

## Dependencies
No dependencies

## Sandbox environment
TODO

### CODE REVIEW

**Review by poyuki — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by wazelin — CHANGES_REQUESTED** [review]

File: locales/ar-arb/messages.po [hunk 1]
```diff
@@ -177,16 +166,14 @@ msgstr "الرجاء تحديد نوع البند أولاً!"
...
 msgid "Preview"
 msgstr "معاينة"
 
-#, tao-public
 msgid "Properties"
 msgstr "خصائص"
 
 msgid "Published \"%1$s\" in language \"%2$s"
-msgstr "تم نشر \"٪ 1 $ s\" باللغة \"٪ 2 $ s"
+msgstr "تم نشر \"%1$s\" باللغة \"%2$s"
```
[the remark this answers is not in the document]
Comment (quintanilhar) [reply locales/ar-arb/messages.po:176]: It seems to be a visual glitch on the GitHub diff feature. Look:

image

This is the output when I copy the symbols from the terminal: https://3v4l.org/SPm4U

[the remark this answers is not in the document]
Comment (quintanilhar) [reply locales/ar-arb/messages.po:176]: The main problem is that we're mixing Arabic chars with Latin chars.

**Review by marpesia — APPROVED** [review]
LGTM

File: locales/ar-arb/messages.po (hunk 1 above)
[the remark this answers is not in the document]
Comment (wazelin) [reply locales/ar-arb/messages.po:176]: True that, I didn't realize it!
The quotes are still a mismatch in the msgstr.
```suggestion
msgstr "تم نشر \"%1$s\" باللغة \"%2$s\""
```

**Review by wazelin — CHANGES_REQUESTED** [review]

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Pull request merged**